python/source/robot_model/bind_model.cpp

```diff
@@ -30,7 +30,23 @@ void model(py::module_& m) {
 
   py::class_<Model> c(m, "Model");
 
-  c.def(py::init<const std::string&, const std::string&>(), "Constructor with robot name and path to URDF file.", "robot_name"_a, "urdf_path"_a);
+  c.def(py::init([](const std::string& robot_name, const std::string& urdf_path, py::object meshloader_callback) {
+  std::function<std::string(const std::string&)> callback_cpp = nullptr;
+  if (!meshloader_callback.is_none()) {
+        callback_cpp = [meshloader_callback](const std::string& package_name) -> std::string {
+              auto result = meshloader_callback(package_name).template cast<std::string>();
+              return result;
+        };
+  }
+  return new Model(robot_name, urdf_path, callback_cpp);
+  }), py::arg("robot_name"), py::arg("urdf_path"), py::arg("meshloader_callback"));
+
+
+  c.def(py::init<const std::string&, const std::string&>(),
```

Comment:
Could you add a short brief to the constructors, like in the old version? Should be possible even if the lambda

Reply:
@yrh012 marked this as resolved but I don't see any annotated documentation yet